### PR TITLE
Create Backend Documentation

### PR DESCRIPTION
**Pull request by salam4**
PMs Please review .

Thanks,
Saqeb

### CODE REVIEW

File: Backend Documentation [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# AssetTracker Backend Technologies                                                                           version 1811.18
+**Reason why AssetTracker has strong Backend development**
+1. *Security* - AssetTracker code needs to be verified and trusted as part of AssetTracker Software, so users do not attempt to bypass the validation and/or security.
+2. *Performance* - AssetTracker is built on do it once and cache computation to avoid duplication. Ability to process heavy computation on fast machines. 
+3. *Capability* - AssetTracker bring dynamic behavior to the table with requiring extensive JavaScript Support.
+
+## AssetTracker Built with Node.JS
```
Comment (AmadoSuarez) [inline Backend Documentation]: We are using Ruby on Rails for the backend.

File: Backend Documentation [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+*Why use Node.JS for AssetTracket?*
```
Comment (AmadoSuarez) [inline Backend Documentation]: Ruby on Rails

**Review by AmadoSuarez — CHANGES_REQUESTED** [review]

File: Backend Documentation (hunk 1 above)
Comment (salam4) [reply Backend Documentation]: I fixed the documentation.

File: Backend Documentation (hunk 2 above)
Comment (salam4) [reply Backend Documentation]: I fixed the documentation

**Review by AmadoSuarez — APPROVED** [review]

**Comment by AmadoSuarez** [comment]
LGTM! 🚢 Merge it!

**Pull request merged**